command/command.go

```diff
@@ -15,17 +15,16 @@ import (
 var (
 	// Could be modified in tests.
 	maxGUBatchSize       = 500
-	maxClientObjectQuota = 85000
+	maxClientObjectQuota = 60000
 )
 
 const (
-	storeBirthday              string = "1"
-	maxCommitBatchSize         int32  = 90
-	setSyncPollInterval        int32  = 30
-	nigoriTypeID               int32  = 47745
-	deviceInfoTypeID           int    = 154522
-	historyTypeID              int    = 963985
```

Comment:
Just checking - this was intentionally removed, right? seems to use `datastore.HistoryTypeID` instead

Reply:
correct